aptos-move/framework/aptos-framework/sources/aggregator_v2/aggregator_v2.move

```diff
@@ -0,0 +1,73 @@
+/// This module provides an interface for aggregators (version 2).
+/// Only skeleton - for AggregagtorSnapshot - is provided at this time,
+/// to allow transition of usages.
+module aptos_framework::aggregator_v2 {
+    use std::string::String;
+
+    /// The generic type supplied to the aggregator snapshot is not supported
+    const EUNSUPPORTED_AGGREGATOR_SNAPSHOT_TYPE: u64 = 5;
```

Comment:
why 5?

Reply:
We have been using 1-4 error codes for overflows and underflows, etc. Eventually they will need to be included in the same file.